packages/cli/src/lib/configureStorybook.js

```diff
@@ -0,0 +1,70 @@
+import fs from 'fs-extra'
+
+import { getPaths } from '.'
+
+/**
+ * Configure Storybook for the given template by creating a custom preview config
+ */
+export default function configureStorybook({ force }, newStorybookPreview) {
+  const storybookPreviewConfigPath = getPaths().web.storybookPreviewConfig
+
+  let storybookPreviewConfig
+  /**
+   *  Check if storybookPreviewConfigPath already exists.
+   *  Merge both files if it does.
+   *  By removing import react and export decorator from new config
+   *  And put new config inside current config after last import
+   */
+  if (fs.existsSync(storybookPreviewConfigPath)) {
+    if (force) {
+      fs.unlinkSync(storybookPreviewConfigPath)
+      storybookPreviewConfig = newStorybookPreview
+    } else {
+      const currentConfig = fs
+        .readFileSync(storybookPreviewConfigPath)
+        .toString()
+
+      const newDecoratorsName = newStorybookPreview.match(
+        /export const decorators = \[(.*?)\]/
+      )[1]
+
+      const currentDecoratorsName = currentConfig.match(
+        /export const decorators = \[(.*?)\]/
+      )[1]
+
+      const decoratorsExport = `export const decorators = [${currentDecoratorsName}, ${newDecoratorsName}]`
+
+      const insideNewStorybookConfigWithoutReactAndDecoration =
+        newStorybookPreview
+          .replace(/import *. as React from 'react'/, '')
```

Comment:
```suggestion
          .replace(/import *. as React from 'react'/, '')
```

Sorry I didn't see this earlier. Did you mean to make that `.*` (instead of `*.`)?

Reply:
I test the regex, I think is right, but I will double-check

Reply:
Yeah, it matches "space" zero or more times, then "any character", then "space", etc. So it works. I just wasn't sure if that's what you were going for :)

Reply:
You probably want `/import \* as React from 'react'/`

Reply:
Oh! yeah right, I had tried this and didn't work, but I'm not the best at regex, so I will test again.

Reply:
https://regexr.com/6if2h